lib/rescue_registry/show_exceptions.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# This is for use in non-Rails Rack apps. Rails apps will handle exceptions with
+# ActionDispatch::DebugExceptions and ActionDispatch::ShowExceptions
+module RescueRegistry
+  class ShowExceptions
+    # Unfortunately, Rail's nice mime types are in ActionDispatch which we'd rather not require
+    CONTENT_TYPES = {
+      jsonapi: "application/vnd.api+json",
+      json:    "application/json",
+      xml:     ["application/xml", "text/xml"],
+      plain:   "text/plain"
+    }
+
+    # Match Rail's Mime::Type API on a basic level
+    MimeType = Struct.new(:to_sym, :to_s)
```

Comment:
This feels a little iffy, but I don't have a better way to do this without depending upon Action Dispatch.

Reply:
Works for me, having this class at all is 💯